src/core_plugins/elasticsearch/lib/create_admin_cluster.js

```diff
@@ -0,0 +1,19 @@
+export default function (server) {
+  const config = server.config();
+  const Logger = server.plugins.elasticsearch.ElasticsearchClientLogging;
```

Comment:
How about destructuring this rather than renaming it?  One less mental exercise to go through when reading the class definition.

```js
const { ElasticsearchClientLogging } = server.plugins.elasticsearch;

class AdminClientLogging extends ElasticsearchClientLogging
```